src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -114,6 +114,12 @@ public static void MapCancelButtonColor(ISearchBarHandler handler, ISearchBar se
 			handler.PlatformView?.UpdateCancelButtonColor(searchBar);
 		}
 
+		static void MapFocus(ISearchBarHandler handler, ISearchBar searchBar, object? args)
+		{
+			ViewHandler.MapFocus(handler, searchBar, args);
```

Comment:
I think we want to tie this in with the existing extension method. 

The way `UpdateFocus` is written in this PR is a little confusing because if I only call `UpdateFocus` it doesn't actually Update the focus

Instead of having the handler here call up to the `ViewHandler` probably just handle this inside the extension method. 

Maybe something like this

![image](https://user-images.githubusercontent.com/5375137/217385795-f86e7e1b-7daa-4dae-9b4b-3163549687bb.png)

Reply:
If we handle this from the ViewExtensions (as I understand you are suggesting), how would we call into it from the mapper? I'm having issues passing in the request from there.

Also, what is the purpose of adding the `Action focusRequested` and invoking it? How does that tie in? I'm not seeing that RequestFocus() method get hit